builtin/game/register.lua

```diff
@@ -368,6 +368,12 @@ core.register_node(":ignore", {
 	air_equivalent = true,
 	drop = "",
 	groups = {not_in_creative_inventory=1},
+	-- Prevent placement of ignore (to prevent engine error message spam in logs)
+	-- and destroy itemstack.
+	on_place = function(itemstack, placer, pointed_thing)
+		minetest.chat_send_player(placer:get_player_name(), minetest.colorize("#FF0000", "One does not simply place 'ignore' nodes!"))
```

Comment:
Line still needs a break. It's way over the 80 allowed characters.

Reply:
OK.